controllers/extension-certificate-service/charts/cert-broker/values.yaml

```diff
@@ -0,0 +1,31 @@
+# Default values for cert-broker.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicas: 1
+
+images:
+  cert-broker: eu.gcr.io/gardener-project/gardener/cert-broker:0.2.0
```

Comment:
I'd not put a concrete image here to avoid confusion where the actually used tag is coming from (should come from image vector), similar to https://github.com/gardener/gardener/blob/master/charts/seed-controlplane/charts/cert-broker/values.yaml#L8.